OTAnalytics/plugin_ui/customtkinter_gui/toplevel_template.py

```diff
@@ -1,45 +1,27 @@
 from abc import ABC, abstractmethod
 from typing import Any
 
-from customtkinter import CTkButton, CTkFrame, CTkToplevel
+from customtkinter import CTkToplevel
 
-from OTAnalytics.application.config import ON_MAC
-from OTAnalytics.plugin_ui.customtkinter_gui.constants import PADX, PADY, tk_events
+from OTAnalytics.plugin_ui.customtkinter_gui.constants import tk_events
 
 
 class ToplevelTemplate(CTkToplevel, ABC):
     def __init__(
         self,
+        title: str,
         initial_position: tuple[int, int],
         **kwargs: Any,
     ) -> None:
         super().__init__(**kwargs)
-        self._get_frame_ok_cancel()
+        self.title(title)
         self._set_ok_cancel_bindings()
         self._set_initial_position(initial_position)
-        self._set_focus()
 
     def _set_initial_position(self, initial_position: tuple[int, int]) -> None:
         x, y = initial_position
         self.geometry(f"+{x+10}+{y+10}")
 
-    def _get_frame_ok_cancel(self) -> None:
-        self.frame_ok_cancel = CTkFrame(master=self)
-        self.button_ok = CTkButton(
-            master=self.frame_ok_cancel, text="Ok", command=self._on_ok
-        )
-        self.button_cancel = CTkButton(
-            master=self.frame_ok_cancel, text="Cancel", command=self._on_cancel
-        )
-        if ON_MAC:
-            ok_column = 1
-            cancel_column = 0
-        else:
-            ok_column = 0
-            cancel_column = 1
-        self.button_ok.grid(row=0, column=ok_column, padx=PADX, pady=PADY)
-        self.button_cancel.grid(row=0, column=cancel_column, padx=PADX, pady=PADY)
-
```

Comment:
Why is the Ok/Cancel frame removed from the template?
This results in duplicated code, because every implementation of `ToplevelTemplate` has to instantiate the frame_ok_cancel. It might be easier to have a "content" frame in `ToplevelTemplate` which is filled with elements from derived classes.

Reply:
Done :)